From a33a53df476134d0bfb823d2ea8833785d9af663 Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Wed, 25 Apr 2018 15:38:11 +0800
Subject: [PATCH 463/767] staging: fsl-dpaa2/eth: change mc_command in
 fsl_mc_command

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpni.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpni.c b/drivers/staging/fsl-dpaa2/ethernet/dpni.c
index f9ddaae4693d..da7b904a6da2 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpni.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpni.c
@@ -934,7 +934,7 @@ int dpni_set_tx_shaping(struct fsl_mc_io *mc_io,
 			u16 token,
 			const struct dpni_tx_shaping_cfg *tx_shaper)
 {
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 	struct dpni_cmd_set_tx_shaping *cmd_params;
 
 	/* prepare command */
@@ -1399,7 +1399,7 @@ int dpni_set_qos_table(struct fsl_mc_io *mc_io,
 		       const struct dpni_qos_tbl_cfg *cfg)
 {
 	struct dpni_cmd_set_qos_table *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_SET_QOS_TBL,
@@ -1437,7 +1437,7 @@ int dpni_add_qos_entry(struct fsl_mc_io *mc_io,
 		       u16 index)
 {
 	struct dpni_cmd_add_qos_entry *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_ADD_QOS_ENT,
@@ -1469,7 +1469,7 @@ int dpni_remove_qos_entry(struct fsl_mc_io *mc_io,
 			  const struct dpni_rule_cfg *cfg)
 {
 	struct dpni_cmd_remove_qos_entry *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_REMOVE_QOS_ENT,
@@ -1508,7 +1508,7 @@ int dpni_add_fs_entry(struct fsl_mc_io *mc_io,
 		      const struct dpni_fs_action_cfg *action)
 {
 	struct dpni_cmd_add_fs_entry *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_ADD_FS_ENT,
@@ -1546,7 +1546,7 @@ int dpni_remove_fs_entry(struct fsl_mc_io *mc_io,
 			 const struct dpni_rule_cfg *cfg)
 {
 	struct dpni_cmd_remove_fs_entry *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_REMOVE_FS_ENT,
@@ -1583,7 +1583,7 @@ int dpni_set_congestion_notification(
 			const struct dpni_congestion_notification_cfg *cfg)
 {
 	struct dpni_cmd_set_congestion_notification *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(
@@ -1630,7 +1630,7 @@ int dpni_get_congestion_notification(
 {
 	struct dpni_rsp_get_congestion_notification *rsp_params;
 	struct dpni_cmd_get_congestion_notification *cmd_params;
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 	int err;
 
 	/* prepare command */
@@ -1826,7 +1826,7 @@ int dpni_reset_statistics(struct fsl_mc_io *mc_io,
 			  u32 cmd_flags,
 			  u16 token)
 {
-	struct mc_command cmd = { 0 };
+	struct fsl_mc_command cmd = { 0 };
 
 	/* prepare command */
 	cmd.header = mc_encode_cmd_header(DPNI_CMDID_RESET_STATISTICS,
-- 
2.17.0

